R/demean.R

```diff
@@ -334,10 +334,6 @@ degroup <- function(x,
     )
   }
 
-  # make sure we have only valid variables
-  select <- intersect(colnames(x), select)
-  by <- intersect(colnames(x), by)
-
```

Comment:
@etiennebacher Since we throw an error now, we no longer need these code lines. However, `intersect()` could change the order of elements, that's why the snapshot test failed.

Former column order: Sepal.Length_between, Species_between, binary_between...
Now: Sepal.Length_between, binary_between, Species_between

However, the new behaviour aligns with variable order in `select`, so it's ok.

Reply:
Ok, I think the docs also need to be updated (note below params)

Reply:
What do you mean?

Reply:
Ah, I see.

Reply:
Ah, I see